templates/hooks/alias/sdk_update_post_build_request.go.tpl

```diff
@@ -0,0 +1 @@
+    input.AliasName = desired.ko.Spec.Name
```

Comment:
So, unless I'm mistaken, by setting this, you basically guarantee that you won't need to paginate through more than a single result, no?